flight/mcu/lib/teensyShared/BuzzerController/BuzzerControl.hpp

```diff
@@ -0,0 +1,446 @@
+#ifndef BUZZER_CONTROL_H
+#define BUZZER_CONTROL_H
+
+#include "dataLog.h"
+#include "acShared.h"
+
+// the following table should be generated by beep_fsm_gen.py
+// TODO this doesn't need a full fsm, we can get away with a struct of a sequence and a "should_loop" parameter
+// TODO it would make the code a lot simpler and more comprehensible
+/* ======= AUTOGENERATED ======= */
+
+enum BuzzerState {
```

Comment:
I see the giant "AUTOGENERATED" comment, but that still doesn't let me know why we this phat enum over here. Why can't we get away with one per buzzer sequence?